docs/quickstart.md

```diff
@@ -0,0 +1,249 @@
+- :fontawesome-solid-microchip: `16 Core` Machine Types will consume `16` hours for every hour of runtime
+
+Some Arista Community Labs make use of larger machine types, such as the :fontawesome-solid-microchip: `16 Core` option.
+
+??? question "My GitHub Account doesn't have access to a larger :fontawesome-solid-microchip: `8 Core` or :fontawesome-solid-microchip: `16 Core` machine types?"
+    By default, some larger machine types may not be available for use. In order to resolve this, a ticket can be opened with [GitHub support](https://support.github.com/contact?source=subtitle&tags=rr-general-technical) requesting access to these larger machine types.
+
+    Listed below is a template that can be used for this request:
+
+    ```yaml
+    Hello - Can the 8-core and 16-core codespace machine types please be enabled for my account?
+    The default 2-core and 4-core machines lack the necessary resources for my use cases.
+    If additional information is needed, please let me know. Thanks!
+    ```
+
+    Once completed, select **Create a Ticket**
+
+## Starting the Lab
+
+!!! note
+    Throughout this guide, the [EVPN Domain A](https://codespaces.new/aristanetworks/aclabs/tree/main?quickstart=1&devcontainer_path=.devcontainer%2Ftechlib-vxlan-domain-a%2Fdevcontainer.json) lab associated with the [EVPN/VXLAN Deployment Guide](https://tech-library.arista.com/data_center/evpnvxlan/deployment_guide/) on [Arista's Tech Library](https://tech-library.arista.com) will be used as a reference point. All community labs hosted via Codespaces will follow the same process.
+
+Once a lab has been launched via it's respective 'Start Lab' button, a `Create Codespace` window will be opened via a web browser:
+
+<figure markdown>
+![Create Codespace](assets/img/aclabs-quickstart-1.png "Create Codespace"){ width=500px }
+<figcaption> Create a Codespace - Enter our user token </figcaption>
+</figure>
+
+In the `ARTOKEN` field, paste the user token copied from the arista.com user profile. This is a one time requirement, and will be saved for all subsequent Arista Community Lab deployments.
+
+??? question "Where is the token saved? :thinking:"
+    Once entered, the token is saved as a GitHub Codespaces `Secret`. This can be viewed via the [Codespaces section of GitHub account settings](https://github.com/settings/codespaces).
+
+??? note "Watch the expiration date! :hourglass_flowing_sand:"
+    User tokens on arista.com have an expiration date listed in the `Token Valid Till` field in the `Portal Access` section of the user profile.
+If the token has expired, click Regenerate Token to create a new one. A token expires one year after it was generated; the user cannot modify this value.
+
+    If the token has expired, simply click `Regenerate Token` to create a new one. A token expires one year after it was generated; This value cannot be modified by the user.
```

Comment:
If this has to be highlighted, an admonition would be better. Currently it's a long gray line that is hard to scroll.

Reply:
Good catch - this was a mistake in formatting!

Reply:
Fixed. Thanks again